docs/manual/java/concepts/code/lagom-immutables.sbt

```diff
@@ -1,4 +1,10 @@
 //#lagom-immutables
 lazy val usersApi = (project in file("usersApi"))
   .settings(libraryDependencies += lagomJavadslImmutables)
-//#lagom-immutables
\ No newline at end of file
+//#lagom-immutables
+
+//#lagom-immutables-lombok
+val lombok = "org.projectlombok" % "lombok" % "1.16.12"
+lazy val usersApi = (project in file("usersApi"))
+  .settings(libraryDependencies += lombok)
+//#lagom-immutables-lombok
```

Comment:
This produced a compile error since there are two `usersApi` defined in the same file.

Note on style, I think it suffices to update both code snippets to just show:

```scala
libraryDependencies += lombok
```

rather than put it in the context of a full project, an sbt user will very quickly learn how to add settings to a project and I don't think needs the full project context.